tests/mmptest/src/MMPTest.cs

```diff
@@ -1,3 +1,5 @@
+// #define ENABLE_STATIC_REGISTRAR_TESTS https://bugzilla.xamarin.com/show_bug.cgi?id=58629
```

Comment:
@chamons That should become an option inside the test harness, since it's a recurring theme from Apple :(
It also should cover the previous fixes from @rolfbjarne 
OTOH it's good for this _bump PR_, that will unblock the updates. Thanks!